packages/cli/cli/src/commands/generate/generateAPIWorkspace.ts

```diff
@@ -25,7 +25,7 @@ export async function generateWorkspace({
     version: string | undefined;
     groupName: string | undefined;
     shouldLogS3Url: boolean;
-    token: FernToken;
+    token?: FernToken;
```

Comment:
can we type this as FernToken | undefined, so that consumers are forced to provide a value